README.md

```diff
@@ -83,8 +83,15 @@ Any external contribution is welcome and teams mentioned above will help in the
 2. In order to start work, first update branch `main`, and then create a new branch from this branch in which future changes will be placed. We adopted the
    nomenclature for the branch `feature/[task_id]`.
 3. If changes require it, appropriate unit tests should also be included, and an additional case should be attached to the Storybook documentation (in the `.stories.tsx` file).
-4. After the work is completed, create a pull request directed to branch `main`. In pull request, you should call `livechat/design-system` in the reviewers field.
-   The merge template contains a checklist of things that need to be completed to meet the requirements, it will make the work easier for everyone.
+4. After the work is completed, create a pull request directed to branch `main`.
+   1. **Conventional PR Naming**: Ensure your pull request title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) naming pattern `type(scope): description`. This is crucial as, post squash merging, Git will use the PR name as the commit message.
+      <br/>To mark changes as a **breaking change**, include a `!` after the type and scope, followed by a colon and a space, like so: `type(scope)!: description`.
+      <br/>For now we will use the following types:
+      - `feat` - for new features
+      - `fix` - for bug fixes
+      - `docs` - for documentation changes
+   2. **Review**: In pull request, you should call `livechat/design-system` in the reviewers field.
+   3. **Template**: The merge template contains a checklist of things that need to be completed to meet the requirements, it will make the work easier for everyone.
```

Comment:
Some slight changes proposed:

```suggestion
4. Once you've completed your work, initiate a pull request targeted at the `main` branch.
   1. **Conventional PR Naming**: Ensure the title of your pull request adheres to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format: `type(scope): description`. This is imperative because, upon squash merging, Git will utilize the PR title as the commit message.
      - To denote **breaking changes**, append a `!` right after the type and scope, like this: `type(scope)!: description`.
      - Currently, we recognize these types:
          - `feat` - introduces a new feature
          - `fix` - addresses a bug
          - `docs` - pertains to documentation updates
   2. **Review Process**: When creating your pull request, please tag `livechat/design-system` in the reviewers' section.
   3. **PR Template**: Our merge template includes a checklist, ensuring all requirements are met. Adhering to this will streamline the process for everyone.
```